www/class/centreonAuth.SSO.class.php

```diff
@@ -103,7 +102,18 @@ public function __construct(
             # Get configured values
             $clientId = $this->ssoOptions['openid_connect_client_id'];
             $clientSecret = $this->ssoOptions['openid_connect_client_secret'];
-            $redirectNoEncode = $this->ssoOptions['openid_connect_redirect_url'];
+            if (empty($this->ssoOptions['openid_connect_redirect_url'])) {
+                $redirectNoEncode = '{$scheme}://{$hostname}:{$port}'
```

Comment:
The $ character is not required for substitution
```suggestion
                $redirectNoEncode = '{scheme}://{hostname}:{port}'
```